cray/modules/bos/openapi.yaml

```diff
@@ -1211,6 +1211,12 @@ components:
           type: integer
           description: The default maximum number attempts per node for failed actions.
           example: 1
+        max_component_batch_size:
+          type: integer
+          description: The maximum number of components that a BOS operator will process at once. 0 means no limit.
+          example: 1000
+          minimum: 0
+          maximum: 131071
```

Comment:
I just wanted a maximum that was larger than I figured anyone would want to use. I don't like adding stuff without some kind of limits, to avoid the kind of problems we've seen before.